src/components/sys-footerbar.vue

```diff
@@ -269,14 +303,54 @@
     font-size: 0.675rem;
   }
 
+  button {
+    border-radius: 4px;
+    background-color: #6ACAD8;
+    -webkit-font-smoothing: antialiased;
+    align-content: center;
+    align-items: center;
+    appearance: none;
+    border-radius: 0.25em;
+    border: 0.15em solid transparent;
+    cursor: pointer;
+    display: inline-flex;
+    font-style: normal;
+    font-weight: 400;
+    justify-content: center;
+    letter-spacing: 1px;
+    line-height: 24px;
+    text-align: center;
+    text-decoration: none;
+    transition-duration: 100ms;
+    transition-property: background-color, border-color, box-shadow;
+    transition-timing-function: ease;
+    user-select: none;
+    letter-spacing: normal;
+    font-family: var(--font-family-sans);
+    text-transform: uppercase;
+    font-size: 1rem;
+    padding: 0.4em 1.6em;
+  }
```

Comment:
Redefining styles kinda sucks, but we have to do that here. Hopefully down the line we can change the button to blend in a little bit better with the footer.